app/controllers/api/databases.php

```diff
@@ -2877,9 +2878,6 @@ function updateAttribute(
             'size' => 0
         ];
 
```

Comment:
Let's validate here that lengths array is not longer than the attributes array, otherwise throw a new exception type: `Exception::INDEX_LENGTHS_INVALID` and in errors.php set the message to `Invalid index lengths. Count of lengths must be equal or less than the number of attributes.`